doc/content/the-things-stack/cloud/subscribe-tts-via-aws-marketplace/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Subscription via AWS Marketplace"
+description: ""
+weight: 2
+---
+
+This section shows how to subscribe a tenant in {{% tts %}} Cloud deployment via AWS Marketplace.
+
+<!--more-->
+
+## Subscribing {{% tts %}} Cloud tenant
+
+Navigate to [The Things Stack Cloud for LoRaWAN](https://aws.amazon.com/marketplace/pp/prodview-djx65iyt6w5cy) in AWS Marketplace.
+
+Sign in to the AWS account and click on `View purchase options`.
+
+{{< figure src="purchase-option.png" alt="AWS Console shows View purchase option" >}}
+
+You will be navigated to the below page, then click on `Subscribe`.
+
+{{< figure src="subscribe.png" alt="Subscribe option" >}}
+
+Navigate to the `Manage Subscriptions` in the AWS Marketplace.
+
+{{< figure src="manage-subscriptions.png" alt="Manager Subscriptions option" >}}
+
+In {{% tts %}} Cloud for LoRaWAN subscription, select the option `Set up product` at the bottom.
+
+{{< figure src="setup-product.png" alt="Set up product" >}}
+
+You will observe a pop up, i.e., `To continue, set up your account and complete the registration`. Then Click on `Set up your account`.
+
+{{< figure src="setup-account.png" alt="Set up your account" >}}
+
+Fill in all the details on the `Configure your account` page.
+
+{{< figure src="configure-account.png" alt="Configure Account" >}}
```

Comment:
Can you please retake screenshots in a way described in [CONTRIBUTING doc](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/CONTRIBUTING.md)? This one looks off, and the `setup-product.png` above is unequally cropped